developments_build/sql/_status.sql

```diff
@@ -245,34 +249,34 @@ AND a.job_number in (
 	FROM CORR_devdb
 	WHERE 'units_incomplete'=any(x_dcpedited));
 
--- x_inactive
-WITH CORR_target as (
-	SELECT a.job_number, 
-		COALESCE(b.reason, 'NA') as reason,
-        b.edited_date
-	FROM STATUS_devdb a, housing_input_research b	
-	WHERE a.job_number=b.job_number
-    AND b.field = 'x_inactive'
-    AND (upper(a.x_inactive)=upper(b.old_value) 
-        OR (a.x_inactive IS NULL 
-            AND (b.old_value IS NULL 
-            OR b.old_value = 'false')))
-)
-UPDATE CORR_devdb a
-SET x_dcpedited = array_append(x_dcpedited, 'x_inactive'),
-	x_reason = array_append(x_reason, json_build_object(
-		'field', 'x_inactive', 'reason', b.reason, 
-		'edited_date', b.edited_date
-	))
-FROM CORR_target b
-WHERE a.job_number=b.job_number;
+-- -- x_inactive
+-- WITH CORR_target as (
+-- 	SELECT a.job_number, 
+-- 		COALESCE(b.reason, 'NA') as reason,
+--         b.edited_date
+-- 	FROM STATUS_devdb a, housing_input_research b	
+-- 	WHERE a.job_number=b.job_number
+--     AND b.field = 'x_inactive'
+--     AND (upper(a.x_inactive)=upper(b.old_value) 
+--         OR (a.x_inactive IS NULL 
+--             AND (b.old_value IS NULL 
+--             OR b.old_value = 'false')))
+-- )
+-- UPDATE CORR_devdb a
+-- SET x_dcpedited = array_append(x_dcpedited, 'x_inactive'),
+-- 	x_reason = array_append(x_reason, json_build_object(
+-- 		'field', 'x_inactive', 'reason', b.reason, 
+-- 		'edited_date', b.edited_date
+-- 	))
+-- FROM CORR_target b
+-- WHERE a.job_number=b.job_number;
 
-UPDATE STATUS_devdb a
-SET x_inactive = trim(b.new_value)
-FROM housing_input_research b
-WHERE a.job_number=b.job_number
-AND b.field = 'x_inactive'
-AND a.job_number in (
-	SELECT DISTINCT job_number
-	FROM CORR_devdb
-	WHERE 'x_inactive'=any(x_dcpedited));
\ No newline at end of file
+-- UPDATE STATUS_devdb a
+-- SET x_inactive = trim(b.new_value)
+-- FROM housing_input_research b
+-- WHERE a.job_number=b.job_number
+-- AND b.field = 'x_inactive'
+-- AND a.job_number in (
+-- 	SELECT DISTINCT job_number
+-- 	FROM CORR_devdb
+-- 	WHERE 'x_inactive'=any(x_dcpedited));
```

Comment:
Should these be uncommented or removed?

Reply:
unclear, because `x_inactive` needs to be changed in the research table first, and there are also corresponding qaqc checks, so I decided to comment it for now and address it during QAQC